lib/debug_logging/configuration.rb

```diff
@@ -66,6 +66,7 @@ def initialize(**options)
       @add_invocation_id = options.key?(:add_invocation_id) ? options[:add_invocation_id] : true
       @ellipsis = options.key?(:ellipsis) ? options[:ellipsis] : DEFAULT_ELLIPSIS
       @mark_scope_exit = options.key?(:mark_scope_exit) ? options[:mark_scope_exit] : false
+      @active_support_notifications = options.key?(:active_support_notifications) ? options[:active_support_notifications] : false
```

Comment:
This will set the value, but it does not call the method which requires the files.  But this is following the same pattern I used for the benchmarks logic, so I think it is broken there as well.  In other words, these three settings will work when done as line item configs, and will not work as block configs.

It is problematic that we are setting each of these instance variables in two places:
```
@active_support_notifications
@instance_benchmarks
@class_benchmarks
```

We need to refactor so the variable is set by a setter that both locations call, and which also does the requires.

Reply:
Nice catch 👍   I refactored the `initialize` method for the three variables that need to be checked in order to run the requires (`@active_support_notifications`, `@instance_benchmarks`, `@class_benchmarks`), so that the setter is always called in these cases.